### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by GlizzyProductions**
starting the assignment

### CODE REVIEW

**Review by matkatmusic — CHANGES_REQUESTED** [review]
Complete the rest of the assignment

File: main.cpp [hunk 1]
```diff
@@ -1010,14 +884,245 @@ Part 1e - Step 19: Request a review
...
+        void playAudioMessage();
+    };
+    //3 things it can do:
+    //1) break cat food robot
+    void breakCatFoodRobot(CatFoodRobot newRobot);
+    //2) knock over objects
+    void knockOverObjects();
+    //3) scratch visitors
+    void scratchVisitors();
+    CatFoodRobot currentCatFeeder;
```
Comment (matkatmusic) [inline main.cpp]: Cats wouldn't own a `CatFoodRobot`.
a `CatOwner` would. 
remove this member variable
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -1010,14 +884,245 @@ Part 1e - Step 19: Request a review
...
+        void scheduleCheckUp(std::string returnDate, bool sameDoctor);
+    };   
+    //3 things it can do:
+    //1) visit the doctor
+    void visitDoctor(HealthStatus updateHealthStatus);
+    //2) go to sleep
+    void goToSleep(int howLong);
+    //3) donate blood
+    void donateBlood(bool giveLeftArm);
+    HealthStatus annualCheckup;
```
Comment (matkatmusic) [inline main.cpp]: an `AnnualCheckup` would provide a `HealthStatus`.  
but an `AnnualCheckup` is NOT a `HealthStatus`.  
this variable should be called `healthStatus`;
If you wanted this idea to be expressed correctly, you'd have a function like this:
```
HealthStatus goToAnnualPhysical(CollegeStudent resident);
```
where `resident` is obviously a student doctor who hasn't completed their residency yet at a hospital or clinic. 
Code represents real life concepts.  it is important that we choose names and create objects that represent real life and real life concepts.
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -1010,14 +884,245 @@ Part 1e - Step 19: Request a review
...
+    //3 things it can do:
+    //1) select hair color
+    void selectHairColor(std::string colorChoice);
+    //2) increase weight
+    float increaseWeight(int addedPounds); //returns current weight + amount of lbs added.
+    //3) lose limbs
+    void loseLimbs();
+};
+
+struct EducationLevel
```
Comment (matkatmusic) [inline main.cpp]: `EducationLevel` is a single thing.  `Education` encompasses many things.
This class should be called `Education`
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (GlizzyProductions) [reply main.cpp]: I think I understand! So I figured cats have collars so I changed the nested class to CatCollar to make things more realistic.

File: main.cpp (hunk 2 above)
Comment (GlizzyProductions) [reply main.cpp]: point taken, I will keep this in mind moving forward.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**